docs/data_catalog/frontend/data_catalog_assets.mdx

```diff
@@ -308,4 +308,52 @@ Then the user can choose which fields are to be bulk edited: you can select the
   </div>
 </div>
 
-In particular, for each selected field, the user can choose to clear the field value on all those selected assets, or to overwrite the any already existent value with a new one set by the user during the bulk edit flow.
\ No newline at end of file
+In particular, for each selected field, the user can choose to clear the field value on all those selected assets, or to overwrite the any already existent value with a new one set by the user during the bulk edit flow.
+
+### Export metadata
+
+The Data Catalog offers a robust export feature, enabling users to extract metadata from various data assets directly from the user interface. The supported export format is `.csv`, ideal for tabular data and easy integration with tools like Excel, Google Spreadsheets and business intelligence software.
+This feature is particularly useful to share metadata outside the Data Catalog, such as for offline analysis and for third-party stakeholders that may not have access to the Data Catalog instance itself.  
+The export metadata feature is designed to be intuitive and accessible from key navigation points: the search assets page, and within the specific asset detail page.
+The exported `.csv` file is characterized by a standard structure, involving the following information displayed in the file header:
+
+  * **Default asset fields**, useful to univocally identify a specific asset (`Asset ID`,`Asset Type`,`System of Record`,`Catalog Name`,`Schema`,`Table`,`Column`)
+  * **Common standard properties**, `description`, `tags`, as they are in common for all Data Catalog asset types
+  * **Custom properties**, based on the custom properties management and metadata enrichment performed by users. The exported file will include all custom properties valorized in the assets which have been exported. For each custom property, in the `.csv` file is also reported the `custom property type`. This allows users to better understand the context of each custom property in relation to the asset.
+
+:::caution
+*Unknown custom properties* found during the export process will be included in the `.csv` file too, but no `custom property type` will be present for them.
+:::
+
+:::info
+It is no necessary to have populated all the default asset fields for each row of the exported file. It depends on the asset type and on the type of connection provider of the SoR from which the asset belongs.
```

Comment:
Maybe we can reword as follow. Please let me know whether this still convey the intended meaning and is sufficient clear.

```suggestion
:::info
Depending on the asset type, some of the default asset fields may not be available for that specific row.
In addition, `Catalog Name` and `Schema` fields receive a corresponding value in each asset row only for selected connection provider.
```